src/pages/chanyoung/Login/components/LoginForm.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Id from './Id';
+import Pw from './Pw';
+import Button from './Button';
+
+class LoginForm extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: {
+        data: '',
+        isVaild: false,
+      },
+      pw: {
+        data: '',
+        isVaild: false,
+      },
+    };
+  }
+
+  changeIsVaild = (type, data, isVaild) =>
+    this.setState({
+      [type]: { data: data, isVaild: isVaild },
+    });
+
+  render() {
+    const { id, pw } = this.state;
+    const isBtn = id.isVaild && pw.isVaild;
+
+    return (
+      <form
+        className="login"
+        onSubmit={e => {
+          this.props.goToMain(id.data, pw.data, isBtn, e);
```

Comment:
```suggestion
          this.props.goToMain(e, id, pw);
```
위 리뷰와 마찬가지로 id, pw를 이미 넘겨줄 수 있기 때문에 isBtn은 불필요합니다

Reply:
이건 실수로 지우지 않았었네요.. 수정했습니다!